src/main/kotlin/io/emeraldpay/dshackle/upstream/grpc/GrpcUpstreams.kt

```diff
@@ -92,7 +94,14 @@ class GrpcUpstreams(
             chanelBuilder.usePlaintext()
         }
 
-        val client = ReactorBlockchainGrpc.newReactorStub(chanelBuilder.build())
+        val client = ReactorBlockchainGrpc.newReactorStub(chanelBuilder.build()).let {
```

Comment:
I think "let" is excessive construction for this code, maybe we can just use it in method body?